docs/DeveloperGuide.adoc

```diff
@@ -196,18 +196,21 @@ image::ListTagsSdForLogic.png[width="800"]
 === Model component
 
 .Structure of the Model Component
-image::ModelClassDiagram.png[width="800"]
+image::ModelClassDiagram.png[width="400", align="center"]
```

Comment:
Looks too small now compared to the other diagrams

Reply:
It is small so that it can fit in our PPP, I can make it bigger if it's too small in your PPP.

Reply:
Possible you link the image directly in your PPP instead of using the tags? that way you don't have to rely on the sizing as stated in the DG and can resize on your PPP directly.

Reply:
Ok good idea. I'll revert the size in the DG.